### DISCUSSION TITLE
Multi GPU - possible? If so, what is the ETA, budget, guideline? If not, why?

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2564] [Q&A] [answered]
I 've just tested this project and I would like to know if it 's possible to do multiple generation at the same time.
If anyone have some idea on it, don't hesitate to message me.
Thank you.

**Comment by mashb1t** [comment, marked as answer]
Nothing has changed since you've opened the discussions https://github.com/lllyasviel/Fooocus/discussions/2292 and https://github.com/lllyasviel/Fooocus/discussions/2302. The feature currently is in the backlog and it doesn't have a high priority, there also is no timeline.
This project is currently not funded nor supported by anyone and contributors are building new features in their spare time.
Feel free to participate and open a PR!
